app/models/cloud_volume_snapshot.rb

```diff
@@ -15,6 +15,8 @@ class CloudVolumeSnapshot < ApplicationRecord
 
   virtual_total :total_based_volumes, :based_volumes
 
+  validates :description, :length => {:maximum => 50}
```

Comment:
This is a great example, where the description (if it's not a name or identifier), could actually be _really_ large coming from a provider.